rust/src/nfs/types.rs

```diff
@@ -15,6 +15,50 @@
  * 02110-1301, USA.
  */
 
+/* RFC 1094, section '2.2 Server Procedures' */
```

Comment:
I think you should reuse https://github.com/OISF/suricata/pull/14178 commit and build on top of it